lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -1427,6 +1443,23 @@ class NGPHistoryParser {
 
       events.push(basal);
     }
+
+    // Add gaps from beginning of Auto Mode start until first microbolus, or basal event
+    for (const event of this.eventsOfType(NGPHistoryEvent.EVENT_TYPE.CLOSED_LOOP_TRANSITION)) {
+      if (event.transitionValue ===
+        NGPUtil.NGPConstants.CL_TRANSITION_VALUE.CL_INTO_ACTIVE) {
+        const gap = this.getAutoModeStartGap(event);
+        const basal = this.cfg.builder.makeAutomatedBasal()
+          .with_rate(0.0)
```

Comment:
Does this need an annotation, as it's fabricated from the auto-mode start event? For example, we use `medtronic/basal/fabricated-from-schedule` and `tandem/basal/fabricated-from-new-day` for other basals that are fabricated from events, see https://github.com/tidepool-org/tideline/blob/master/js/plot/util/annotations/annotationdefinitions.js

Reply:
Sounds like a "yes" to me 😄